atmo/settings.py

```diff
@@ -367,39 +367,17 @@ class Core(AWS, Celery, Constance, CSP, Configuration):
     def DJANGO_AMAZON_SES_REGION(self):
         return self.AWS_CONFIG['AWS_REGION']
 
-    #: Adds the django-allauth authentication backend.
     AUTHENTICATION_BACKENDS = (
         'django.contrib.auth.backends.ModelBackend',
-        'allauth.account.auth_backends.AuthenticationBackend',
+        'mozilla_django_oidc.auth.OIDCAuthenticationBackend',
         'guardian.backends.ObjectPermissionBackend',
     )
 
-    LOGIN_URL = reverse_lazy('account_login')
-    LOGOUT_URL = reverse_lazy('account_logout')
+    LOGIN_URL = reverse_lazy('oidc_authentication_init')
+    LOGOUT_URL = reverse_lazy('oidc_logout')
     LOGIN_REDIRECT_URL = reverse_lazy('dashboard')
-
-    # django-allauth configuration
-    ACCOUNT_LOGOUT_REDIRECT_URL = LOGIN_REDIRECT_URL
-    ACCOUNT_EMAIL_CONFIRMATION_EXPIRE_DAYS = 7
-    ACCOUNT_EMAIL_SUBJECT_PREFIX = EMAIL_SUBJECT_PREFIX
-    ACCOUNT_EMAIL_REQUIRED = True
-    ACCOUNT_EMAIL_VERIFICATION = 'optional'
-    ACCOUNT_LOGOUT_ON_GET = True
-    ACCOUNT_ADAPTER = 'atmo.users.adapters.AtmoAccountAdapter'
-    ACCOUNT_USERNAME_REQUIRED = False
-
-    SOCIALACCOUNT_ADAPTER = 'atmo.users.adapters.AtmoSocialAccountAdapter'
-    SOCIALACCOUNT_EMAIL_VERIFICATION = 'none'  # no extra verification needed
-    SOCIALACCOUNT_QUERY_EMAIL = True  # needed by the Google provider
-
-    SOCIALACCOUNT_PROVIDERS = {
-        'google': {
-            'HOSTED_DOMAIN': 'mozilla.com',
-            'AUTH_PARAMS': {
-                'prompt': 'select_account',
-            }
-        }
-    }
+    LOGIN_REDIRECT_URL_FAILURE = "/BOGUS"
```

Comment:
If this is really used somewhere, let's just use `'/'` here

Reply:
This was just for testing. It should redirect to "/" but it also does that on success.